legos/cryptocurrency.py

```diff
@@ -32,21 +32,28 @@ def handle(self, message):
         self.reply(message, self.lookup_symbol(query), opts)
 
     def lookup_symbol(self, query):
-        request_url = 'https://api.cryptonator.com/api/ticker/{}-USD'.format(
-                      query)
-        api_response = requests.get(request_url)
+        request_url = 'https://min-api.cryptocompare.com/data/price'
+        params = {}
+        params['fsym'] = query  # fsym, the SYMbol to convert From
+        if query == 'BTC':
+            params['tsyms'] = 'USD'  # tsyms, the SYMbolS to convert To
+        else:
+            params['tsyms'] = 'USD,BTC'  # tsyms, the SYMbolS to convert To
+        api_response = requests.get(request_url, params=params)
         if api_response.status_code == requests.codes.ok:
-            return_val = self._parse_for_ticker(api_response, query)
+            return_val = self._parse_api_response(api_response, query)
         else:
             logger.error('Requests encountered an error.')
             logger.error('''HTTP GET response code:
                         {}'''.format(api_response.status_code))
-            api_response.raise_for_status()
+            return_val = api_response.raise_for_status()
```

Comment:
Since we don't do anything with `return_val` other than return it, we can just skip assignment and return that call directly.